main.nf

```diff
@@ -151,6 +154,11 @@ if( params.gtf ){
     gtf = file(params.gtf)
     if( !gtf.exists() ) exit 1, "GTF file not found: ${params.gtf}."
 }
+bed = false
+if( params.bed ){
+    bed = file(params.bed)
```

Comment:
I've gone off directly referring to files in regular variables. It can cause problems on some computational systems with files being staged. Better to always create a channel instead, then nextflow automatically handles the file staging properly.